flow-typed/platform.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+declare var Intl: Object;
+
+export type FluxStandardAction = {
```

Comment:
I ran into some funky stuff that I *think* was due to using `export type` as opposed to `declare type`... although at this point I'm not 100% sure if that was the actual issue or not (just in case you run into issues with Flow not being able to find custom defined types that are dependencies of other custom defined types)